### PR TITLE
Add a `log.log` host export for basic logging in mappings

### PR DESCRIPTION
**Pull request by Jannis**
Part of resolving #599 and https://github.com/graphprotocol/graph-ts/issues/5.

### CODE REVIEW

**Comment by Jannis** [comment]
I'll add a test for `log.log` tomorrow.

**Comment by Jannis** [comment]
I've come to the conclusion that there's no simple/good way to capture and check the `log.log`, so I'd rather not add a test for it.

File: runtime/wasm/src/asc_abi/class.rs [hunk 1]
```diff
@@ -536,3 +537,24 @@ pub(crate) struct AscBigDecimal {
     // Decimal exponent. This is the opposite of `scale` in rust BigDecimal.
     pub exp: AscPtr<AscBigInt>,
 }
+
+#[repr(C)]
```
Comment (leodasvacas) [inline runtime/wasm/src/asc_abi/class.rs]: This should be `#[repr(u32)]`.

**Review by leoyvens — CHANGES_REQUESTED** [review]
Yay logging! Just a comment.

File: runtime/wasm/src/asc_abi/class.rs (hunk 1 above)
Comment (Jannis) [reply runtime/wasm/src/asc_abi/class.rs]: @leodasvacas Updated to `#[repr(u32)]`!

**Review by leoyvens — APPROVED** [review]

**Pull request merged**